src/components/definition-list/definition-list.style.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/prefer-default-export */
+import styled, { css } from 'styled-components';
+import { baseTheme } from '../../style/themes';
+import StyledButton from '../button/button.style';
+import LinkStyle from '../link/link.style';
+
+const theme = baseTheme;
+
+export const StyledDl = styled.dl`
+    display: inline-flex;
+    height: auto;
+    width: 100%;
+    background-color: transparent;
+    overflow: hidden;
+`;
+
+export const StyledDtDiv = styled.div`
+${({ w, dtTextAlign }) => {
+    return css`
+    text-align: ${dtTextAlign};
+    width: ${w}%;
+  `;
+  }
+}`;
+
+export const StyledDdDiv = styled.div`
+${({ w, ddTextAlign }) => {
+    return css`
+    text-align: ${ddTextAlign};
+    width: ${w - 100}%;
```

Comment:
This is always going to be a negative percentage, bar when `w` is 100 which doesn't seem right

Reply:
You're right. I should have the calculation round the other way. Cheers for that 👍

Reply:
✅